gtsfm/configs/deep_front_end.yaml

```diff
@@ -26,7 +26,7 @@ SceneOptimizer:
         use_outdoor_model: True
 
     verifier:
-      _target_: gtsfm.frontend.verifier.ransac.Ransac
+      _target_: gtsfm.frontend.verifier.gric_verifier.GricVerifier
```

Comment:
this is another change I'm validating; I'll provide dashboard results when they finish.

Reply:
@johnwlambert I think having this change in a separate PR will enable this to merge quickly. We can also analyze the performance of palace dataset with the correct ground truth before changing the verifier.

Reply:
Good idea. I removed that change from this PR.